docs/DeveloperGuide.md

```diff
@@ -850,17 +850,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1. User enters the list command.
+1. User requests to view the list of students.
 2. NUSCoursemates displays a list of all students along with their details.
 
    Use case ends.
 
 **Use Case: UC05 - Deleting a Student**
 
 1. User requests to <u>list persons (UC04)</u>
-2. NUSCoursemates shows a list of persons
+2. NUSCoursemates shows a list of students
```

Comment:
Hi Leon, this LGTM! but perhaps can you add a full stop at the end. Thank you! Also, would appreciate if you could help us add the full stops for use cases 5 and 6, thank you !!

Reply:
ahh I forgot about it, thanks!